tests/integration/test_charm.py

```diff
@@ -71,26 +71,10 @@ async def test_openfga_relation(self, ops_test: OpsTest):
         logger.info("adding openfga postgresql relation")
         await ops_test.model.integrate("openfga-k8s:database", "postgresql-k8s:database")
 
-        await ops_test.model.wait_for_idle(
-            apps=["openfga-k8s"],
-            status="blocked",
-            raise_on_blocked=False,
-            timeout=1200,
-        )
-
-        openfga_unit = ops_test.model.applications["openfga-k8s"].units[0]
-        for i in range(10):
-            action: Action = await openfga_unit.run_action("schema-upgrade")
-            result = await action.wait()
-            logger.info(f"attempt {i} -> action result {result.status} {result.results}")
-            if result.results == {"result": "done", "return-code": 0}:
-                break
-            time.sleep(2)
-
```

Comment:
Why has this been removed?

Reply:
This change was introduced from the Temporal k8s operator side in that we are now relating it to the latest version of the OpenFGA charm, which by design does not require running the schema upgrade action on first deployment. The action only needs to be run when the OpenFGA charm is refreshed.